xa/java/department-helidon-jpa/src/main/java/com/oracle/mtm/sample/data/AccountsService.java

```diff
@@ -45,22 +45,14 @@ public class AccountsService implements IAccountsService {
     @TrmEntityManager
     private Provider<EntityManager> trmEntityManager;
 
-    @Inject
```

Comment:
Have we removed this localEntityManager in Configuration class ?

- In all services

Reply:
Hi not all , as of now we decided to change only 4 application